publish.go

```diff
@@ -40,6 +40,8 @@ type PublishOptions struct {
 	ContentType string
 	// Transient or Persistent
 	DeliveryMode uint8
+	// Expiration time in ms a message will expire from a queue.
+	Expiration		string
```

Comment:
Can we add a link to the description in the comment: https://www.rabbitmq.com/ttl.html#per-message-ttl-in-publishers